AppController/lib/djinn_job_data.rb

```diff
@@ -117,6 +117,22 @@ def method_missing(id, *args, &block)
     super
   end
 
+  # In the process of removing roles, db_slave is no longer added to non
+  # db_master nodes. A node that is a database and is not the db_master is
+  # now considered a db_slave.
+  def is_db_slave?
+    return @jobs.include?('db_slave') ||
```

Comment:
Since older deployments still had the database role, I don't think it's necessary. There's never a case (as far as I know) where db_slave is defined and database is not.